AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/DataStoreModelsVersionTests.swift

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreModelsVersionTests: XCTestCase {
+
+    func testVersionIsEmpty() {
+        let schemaUpdateInvoked = expectation(description: "Version has updated and old database is deleted")
+
+        guard let userDefaults = UserDefaults.init(suiteName: "testVersionIsEmpty") else {
```

Comment:
can you make sure `userDefaults` does not contain a version by either
- optimiscally deleting it, ie. `userDefaults.remove(key: SQLiteStorageEngineAdapter.dbVersionKey)`
- or, checking that userDefaults.string(forKey: SQLiteStorageEngineAdapter.dbVersionKey) is nil

Reply:
I may prefer the first option, changed